app/views/vacancies/_filters.html.haml

```diff
@@ -41,8 +41,13 @@
 
   = form_tag jobs_path, class: 'filters-form', method: :get do
     .govuk-form-group
-      = label_tag 'location', t('jobs.filters.location'), class: 'govuk-label'
-      = search_field_tag :location, location, class: 'govuk-input form-control form-control-block', placeholder: t('jobs.filters.location_hint')
+      .js-location-finder
+        = label_tag 'location', t('jobs.filters.location'), class: 'govuk-label'
+        .js-location-finder__input-container#js-location-finder__input-container
+          = search_field_tag :location, location, class: 'govuk-input form-control form-control-block js-location-finder__input', placeholder: t('jobs.filters.location_hint')
+        .js-location-finder__link
+          or
+          = link_to t('jobs.filters.use_current_location'), '#', id: 'current-location'
```

Comment:
could you add these classes to the link to make the styling of the link better? `govuk-link govuk-link--no-visited-state`